core/cap-0067.md

```diff
@@ -145,14 +151,27 @@ contract: asset, topics: ["clawback", from:Address, sep0011_asset:String], data:
 
 At the moment, if the issuer is the sender in a Stellar Asset Contract `transfer`, the asset will be minted. If the issuer is the recipient, the asset will be burned. The event emitted in both scenarios, however, is the `transfer` event. This CAP changes that behavior to instead emit the `mint`/`burn` event.
 
-### Add a `transfer_with_memo` function to the Stellar Asset Contract
+### Update the `transfer` function to take in a `MuxedAccount` for `from` and `to`
+
+This change will allow the `transfer` event to emit muxed information for exchange support. 
+
+### Emit a map as the `data` field in the `transfer` event if muxed information is being emitted.
 
-Add the following function to the Stellar Asset Contract
-`transfer_with_memo(from: Address, to: Address, amount: i128, from_muxed_id: Option<u64>, to_muxed_id: Option<u64>)` and extend the `transfer` event topics to `["transfer", from: Address, to: Address, sep0011_asset: String, muxed_ids:Vec<Option<U64>>]`.
+The `data` field is currently an integer that represents the `amount`. If no muxed information is being emitted, this will not change. If we're emitting muxed information, then the `data` field on the `transfer` event will be an `SCVal` of type `SCV_MAP`. In
```

Comment:
> If no muxed information is being emitted, this will not change.

Is this temporary (e.g., 1 protocol) or permanent?

Reply:
Permanent

Reply:
Your terms are gross but acceptable